### PR TITLE
Document acceptance tests a lil more

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

File: internal/acceptance/README.md [hunk 1]
```diff
@@ -37,9 +37,18 @@ supported:
 
 These arguments need to be prefixed with `-args` parameter, for example:
 
-    $ go test ./internal/acceptance/ -args -persist -tags=@focus
+    $ go test -tags=acceptance ./internal/acceptance -args -persist -tags=@focus
 
-Depending on your setup Testcontainer's ryuk container migh need to be run as
+Notice that there are two arguments called `-args`. One is for specifying the
```
Comment (simonbaird) [inline internal/acceptance/README.md:42]: `-tags` ?

**Review by simonbaird — APPROVED** [review]
I think there's a typo re args vs tags, otherwise looks great.

**Comment by lcarva** [comment]
> I think there's a typo re args vs tags, otherwise looks great.

Nice catch. Thanks!

**Review by zregvart — APPROVED** [review]
Thanks!

**Pull request merged**